golem-worker-service-base/Cargo.toml

```diff
@@ -76,6 +76,7 @@ tracing-subscriber = { workspace = true }
 url = { workspace = true }
 urlencoding = { workspace = true }
 uuid = { workspace = true }
+anyhow = "1.0.97"
```

Comment:
is this needed still? if so, please add to workspace cargo

Reply:
No i will clean up the dependencies.

Reply:
Excuse this accidental addition. I just disabled this auto update of cargo.toml in IDE when I simply add an import. 
Anyway the dependency is removed :)